src/app/features/stacks-transaction-request/submit-action.tsx

```diff
@@ -8,13 +8,13 @@ import { isEmpty } from '@shared/utils';
 import { useDrawers } from '@app/common/hooks/use-drawers';
 import { LoadingKeys, useLoading } from '@app/common/hooks/use-loading';
 import { useTransactionError } from '@app/features/stacks-transaction-request/hooks/use-transaction-error';
-import { ButtonProps, LeatherButton } from '@app/ui/components/button';
+import { Button, ButtonProps } from '@app/ui/components/button/button';
 
 function BaseConfirmButton(props: ButtonProps): React.JSX.Element {
   return (
-    <LeatherButton fullWidth mt="space.04" type="submit" {...props}>
+    <Button fullWidth mt="space.04" type="submit" {...props}>
```

Comment:
For these one off use cases like `BaseConfirmButton` I think we should get rid of the wrapper component and just use `Button`

Reply:
Yeah it's funny. I am enjoying these global updates / UI migrations as it's a nice chance to reflect on what we have and simplify.